CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## main
+
+## 10.11.0-rc.1 - January 26, 2023
+
+* Improve stability of attribution parsing. ([#1849](https://github.com/mapbox/mapbox-maps-ios/pull/1849))
+* Enable `Expression` to be created without an operator so `clusterProperties` can support advanced use cases. ([#1855](https://github.com/mapbox/mapbox-maps-ios/pull/1855))
+* Update CoreMaps `10.11.0-rc.1` and CommonSDK `23.3.0-rc.1`. ([#1856](https://github.com/mapbox/mapbox-maps-ios/pull/1875))
```

Comment:
this is missing changelog entry for https://github.com/mapbox/mapbox-maps-ios/pull/1828

Reply:
There is no changelog entry in #1828 🙂

Reply:
true, should we add one here?

Reply:
Done